docker/openemr/5.0.2/openemr.conf

```diff
@@ -8,64 +9,68 @@ LoadModule rewrite_module modules/mod_rewrite.so
-		Options Indexes FollowSymLinks
-	        AllowOverride None
-		Require all granted
-	</Directory>
+    ErrorLog ${APACHE_LOG_DIR}/error.log
+    CustomLog ${APACHE_LOG_DIR}/access.log combined
 
- <Directory "/var/www/localhost/htdocs/openemr">
-      AllowOverride FileInfo
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites">
-      AllowOverride None
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites/*/documents">
-      order deny,allow
-      Deny from all
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites/*/edi">
-      order deny,allow
-      Deny from all
-  </Directory>
-  <Directory "/var/www/localhost/htdocs/openemr/sites/*/era">
-      order deny,allow
-      Deny from all
-  </Directory>
+    <Directory /var/www/localhost/htdocs/openemr>
+        # Only allow these HTTP Methods
+        AllowMethods GET POST HEAD OPTIONS
+        # No indexes anywhere
+        Options -Indexes
+        Require all granted
+    </Directory>
 
-	# Available loglevels: trace8, ..., trace1, debug, info, notice, warn,
-	# error, crit, alert, emerg.
-	# It is also possible to configure the loglevel for particular
-	# modules, e.g.
-	#LogLevel info ssl:warn
-	ErrorLog ${APACHE_LOG_DIR}/error.log
-	CustomLog ${APACHE_LOG_DIR}/access.log combined
+    <Directory "/var/www/localhost/htdocs/openemr">
+         # Keeping this it might do something important
+         AllowOverride FileInfo
+     </Directory>
 
-<VirtualHost *:80>
+     <Directory "/var/www/localhost/htdocs/openemr/sites">
+         AllowOverride None
+     </Directory>
 
-</VirtualHost>
-        <VirtualHost _default_:443>
+     <Directory "/var/www/localhost/htdocs/openemr/sites/*/documents">
+         Require all denied
+     </Directory>
+
+     <Directory "/var/www/localhost/htdocs/openemr/sites/*/edi">
+         Require all denied
+     </Directory>
 
-                #   SSL Engine Switch:
-                #   Enable/Disable SSL for this virtual host.
-                SSLEngine on
-				
-				SSLCipherSuite HIGH:!MEDIUM:!aNULL:!MD5:!RC4
-				SSLProtocol -ALL +TLSv1.2
-                #   SSLCertificateFile directive is needed.
-                SSLCertificateFile      /etc/ssl/certs/webserver.cert.pem
-                SSLCertificateKeyFile /etc/ssl/private/webserver.key.pem
+     <Directory "/var/www/localhost/htdocs/openemr/sites/*/era">
+         Require all denied
+     </Directory>
 
-                #SSLOptions +FakeBasicAuth +ExportCertData +StrictRequire
-                <FilesMatch "\.(cgi|shtml|phtml|php)$">
-                                SSLOptions +StdEnvVars
-                </FilesMatch>
-                <Directory /usr/lib/cgi-bin>
-                                SSLOptions +StdEnvVars
-                </Directory>
 
-        </VirtualHost>
+<VirtualHost *:80>
+        #RewriteEngine On
+        #RewriteCond %{HTTPS} off
+        #RewriteRule (.*) https://%{HTTP_HOST}/$1 [R,L]
+</VirtualHost>
```

Comment:
Can this block be removed since all commented out?

Reply:
No should not be removed.  If they want to do SSL Redirect later all they have to do is uncomment it and instantly they have it.

Reply:
Sounds very reasonable. Would it make sense to comment out the VirtualHost lines above and below it and also add a comment above it to uncomment the code if wish to redirect all http to httpd?

Reply:
TODO